scripts/bert/run_pretraining.py

```diff
@@ -326,7 +338,8 @@ def train(data_train, model, nsp_loss, mlm_loss, vocab_size, ctx, store):
     fp16_trainer = FP16Trainer(trainer, dynamic_loss_scale=dynamic_loss_scale)
 
     if args.ckpt_dir and args.start_step:
-        trainer.load_states(os.path.join(args.ckpt_dir, '%07d.states'%args.start_step))
+        logging.info('Loading checkpoint from %s', args.ckpt_dir)
```

Comment:
Thanks for adding the log! I was about to add this, too.

What about
```
state_path = os.path.join(args.ckpt_dir, '%07d.states'%args.start_step)
logging.info('Loading trainer state from %s', state_path)
trainer.load_states(state_path)
```

```

Reply:
Looks good, will add.